app/src/routes/[projectId]/pull/[number]/+page.svelte

```diff
@@ -6,24 +6,20 @@
 	// It may also display details about a cooresponding pr if they exist
 	import FullviewLoading from '$lib/components/FullviewLoading.svelte';
 	import PullRequestPreview from '$lib/components/PullRequestPreview.svelte';
-	import { GitHubService } from '$lib/github/service';
-	import { getContext } from '$lib/utils/context';
-	import { map } from 'rxjs';
+	import { getHostedGitListingServiceStore } from '$lib/hostedServices/interface/hostedGitListingService';
 	import { page } from '$app/stores';
 
-	const githubService = getContext(GitHubService);
-
-	$: pr$ = githubService.prs$?.pipe(
-		map((prs) => prs.find((b) => b.number.toString() === $page.params.number))
-	);
+	const githubService = getHostedGitListingServiceStore();
+	$: prs = $githubService?.prs;
+	$: pr = $prs?.find((b) => b.number.toString() === $page.params.number);
```

Comment:
Was there a particular reason you didnt migrate these?

Reply:
I'm actually getting rid of that page in another PR so don't worry about it

Reply:
Just that it wasn't necessary given my change, I migrated the other ones because I wanted to use runes mode for new code. But, I might as well. :)